packages/block-library/src/columns/index.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Server-side rendering of the `core/columns` block.
```

Comment:
I was a bit stuck on this one, because we need to handle published blocks that haven't yet been migrated to the new markup. I'd prefer if we can avoid the server rendering, but unfortunately couldn't quite think of an alternative to ensure blocks out in the wild don't regress 🤔

Reply:
Hmm, it should work fine with just a deprecation, and perhaps some adjustments in the editor logic for converting legacy `%` units to `fr` and stuff like that. Any specific examples of regressions you detected?

Reply:
If I remember correctly, the steps were:

* Prior to this PR, publish an existing columns block with some custom widths
* Apply this PR
* Before going to the editor, view the columns block on the front end of the site, and notice that the styling no longer works, because we don't _yet_ have the `grid-template-columns` attribute in the block markup.

Once the block is opened in the editor, and the markup has been migrated, it's fine, it was just about trying to ensure that we either have CSS that can target both versions of the markup (which this PR doesn't have because flex-basis doesn't do anything when we're using CSS grid), or use server-rendering to get it in there.

Happy to take another look, though!